src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -685,6 +687,35 @@ bool layout_optimizer::convolution_b_fs_yx_fsv16_opt(const layout& input_layout,
     return false;
 }
 
+static bool has_rank3_mvn_user(const program_node& node, size_t cur_depth, size_t max_depth, uint64_t reorder_size_threshold = 0) {
```

Comment:
Actually, this function `has_rank3_mvn_user` does not check the mvn's input rank. What do you think of renaming it as `has_reorder_before_mvn'?

Reply:
Okay, applied.